recipes/dp-accounting/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dp-accounting" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/dp-accounting-{{ version }}.tar.gz
+  sha256: bf7ffc4b8048a0bc44c3601a263a23dc6c4c37bed348f1dfe378eb808ceb3419
+  patches:
+    - set_version.patch
+    - set_requirements.patch
```

Comment:
If you can, please open issues upstream for the missing files and add a comment here so we can track the evolution of these patches.